locale/core.yml

```diff
@@ -217,6 +219,41 @@ core:
       remove_button: => core.ref.remove
       upload_button: Choose an Image...
 
+    users:
+      title: => core.ref.users
+      description: A paginated list of all users on your forum.
+
+      total_users: "Total users: {count}"
+
+      grid:
+        invalid_column_content: Invalid
+
+        default_columns:
```

Comment:
Some of these are columns, others are default column values. Not sure how I feel about the organization.